src/routes/github/collaborator.ts

```diff
@@ -4,17 +4,17 @@ import * as config from '../../config';
 
 import { checkValidations } from '../../middleware/validation.middleware';
 import { authentication } from '../../middleware/authentication.middleware';
-import { Collaborator } from '../../validation/github/collaborator.validation';
+import { collaborator } from '../../validation/github/collaborator.validation';
 import { get, getById, post, postById, removeById } from '../../controller/github/collaborator.controller';
 
 const CollaboratorRouter = Router();
```

Comment:
Very small one, it should be `collaboratorRouter`

Reply:
You are a wizard as spotting these minor mistakes Mo, good stuff 🔥